rena/scripting/Examples/P300SpellerDemo/P300Speller_params.py

```diff
@@ -0,0 +1,89 @@
+
+# P300SpellerGameStateControlMarker
+START_TRAINING_MARKER = 1
+END_TRAINING_MARKER = 2
+START_TESTING_MARKER = 3
+END_TESTING_MARKER = 4
+InterruptExperimentMarker = -1
+
+# P300SpellerTrailStartEndMarker
+TRAIL_START_MARKER = 1
+TRAIL_END_MARKER = 2
+
+# P300SpellerStartFlashingMarker
+FLASHING_MARKER = 1
+
+# P300SpellerFlashingRowOrColumMarker
+ROW_FLASHING_MARKER = 1
+COL_FLASHING_MARKER = 2
+
+# P300SpellerTargetNonTargetMarker
+NONTARGET_MARKER = 1
+TARGET_MARKER = 2
+
+##########
+TRAIN_RESPONSE_MARKER = 1
+TEST_RESPONSE_MARKER = 2
+
+###########################################
+IDLE_STATE = 0
+RECORDING_STATE = 1
+TRAINING_STATE = 99
+TESTING_STATE = 100
+############################################
+
+EEG_SAMPLING_RATE = 250.0
+
+Time_Window = 1.1  # second
```

Comment:
I suggest put 'second' in the variable name so it becomes "time_window_seconds"

Reply:
Fixed